src/scripts/convert_lovd2.php

```diff
@@ -0,0 +1,1312 @@
+            $aVOTRecord['position_c_start'] =   $oResponse->startmain;
+            $aVOTRecord['position_c_start_intron'] =    $oResponse->startoffset;
+            $aVOTRecord['position_c_end'] =             $oResponse->endmain;
+            $aVOTRecord['position_c_end_intron'] =      $oResponse->endoffset;
+        }
+
+        // Call mutalyzer's numberConversion to get VariantOnGenome/DNA
+        $aNumberConvArgs = array(
+            'build' => $_CONF['refseq_build'],
+            'gene' => $zTranscript['geneid'],
+            'variant' => $zTranscript['id_ncbi'] . ':' . $sVariant);
+        $sNumberConvURL = str_replace('/services', '', $_CONF['mutalyzer_soap_url']);
+        $sNumberConvURL .= '/json/numberConversion?' . http_build_query($aNumberConvArgs);
+        $oResponse = lovd_callJSONService($sNumberConvURL);
+        if ($oResponse && !isset($oResponse->errorcode) && !isset($oResponse->faultcode) &&
+            count($oResponse) > 0 && !empty($oResponse[0])) {
+            $oResponseFields = explode(':', $oResponse[0], 2);
+            $aVOGRecord['VariantOnGenome/DNA'] = $oResponseFields[1];
+        }
+
+        $aVOGRecords[$sRecordID] = $aVOGRecord;
+        $aVOTRecords[$sRecordID] = $aVOTRecord;
+    }
+    return array(
+        $aVOGRecords,
+        $aVOTRecords,
+        $aDiseases,
+        $aIndividuals,
+        $aIndividuals2Diseases,
+        $aScreenings,
+        $aScreening2Genes,
+        $aScreening2Variants,
+        $aPhenotypes);
+}
+
+
+
+
+
+function lovd_setUserIDSettings ($sFixedSubmitterIDInput, $sSubmitterTranslationTableInput,
+                                 $sFixedCuratorIDInput, $sCuratorTranslationTableInput)
+{
+    // Validate form settings for handling user IDs. Calls lovd_errorAdd() when
+    // something went wrong. User ID settings are interpreted as follows:
+    // $sFixedUserIDInput:          Default user ID, should be integer
+    //                              referring to existing user.
+    // $sUserTranslationTableInput: Textual translation table, where every line
+    //                              contains two integers separated by
+    //                              whitespace. The first int is the LOVD2 user
+    //                              ID value to translate, the second int is
+    //                              the LOVD3 user ID value to which is being
+    //                              translated.
+
+    global $sFixedSubmitterID, $aSubmitterTranslationTable, $sFixedCuratorID,
+           $aCuratorTranslationTable;
+
+    $aFixedUserInfos = array(
+        array('submitter', 'submitterid_fixed', 'sFixedSubmitterID', $sFixedSubmitterIDInput),
+        array('curator', 'curatorid_fixed', 'sFixedCuratorID', $sFixedCuratorIDInput),
+    );
+
+    foreach ($aFixedUserInfos as $aFixedUserInfo) {
+        list($sIDtype, $sFormField, $sGlobalVar, $sInput) = $aFixedUserInfo;
+        if (ctype_digit($sInput)) {
+            $GLOBALS[$sGlobalVar] = $sInput;
```

Comment:
I didn't want to duplicate the code for setting the variables for submitter and curator (fixed IDs and translation tables)